proto/redpanda/api/dataplane/v1alpha2/pipeline.proto

```diff
@@ -381,7 +381,7 @@ service PipelineService {
     };
     option (redpanda.api.auth.v1.authorization) = {
       required_permission: PERMISSION_VIEW
-      api: API_UNSPECIFIED
+      api: API_CONSOLE
```

Comment:
I believe all of these pipeline protos would be "REDPANDA_CONNECT" and not Console because that's the upstream API we may want to have user impersonation for at some point. Since we do not even have authentication for RP Connect at the moment it doesn't matter though

Reply:
Yup was curious about this... Changed the Pipeline API type to `API_REDPANDA_CONNECT`.